libpromises/systype.c

```diff
@@ -100,7 +103,11 @@ const char *const VPSOPTS[] =
     [PLATFORM_CONTEXT_BUSYBOX] = "",                  /* linux / busybox */
     [PLATFORM_CONTEXT_SOLARIS] = "auxww",     /* solaris >= 11 */
     [PLATFORM_CONTEXT_SUN_SOLARIS] = "auxww", /* solaris < 11 */
-    [PLATFORM_CONTEXT_FREEBSD] = "auxw -J 0",              /* freebsd */
+#if __FreeBSD_version >= 903000
+    [PLATFORM_CONTEXT_FREEBSD] = "auxw -J 0",              /* freebsd 9.3 and newer */
+#else
+    [PLATFORM_CONTEXT_FREEBSD] = "auxw",              /* freebsd 9.2 and older*/
+#endif
```

Comment:
You will have to surround this with `#if defined __FreeBSD__` as well, otherwise the other platforms will complain about comparison with an unknown macro.